src/test/java/com/ylysenkova/movieland/web/controller/MovieControllerTest.java

```diff
@@ -137,6 +135,43 @@ public void getMovieByGenreId() throws Exception {
                 .andExpect(jsonPath("$[0].price", is(movie.getPrice())));
     }
 
+    @Test
+    public void getMovieById() throws Exception {
+
+        Movie movie = new Movie();
+        List<Country> countries = new ArrayList<>();
+        List<Genre> genres = new ArrayList<>();
+        List<Review> reviews = new ArrayList<>();
+        countries.add(new Country("USA"));
+        genres.add(new Genre(2,"Horror"));
+        reviews.add(new Review(4, "tttt"));
+        movie.setId(9);
+        movie.setNameRussian("ggg");
+        movie.setNameNative("roro");
+        movie.setYearOfRelease(2000);
+        movie.setRating(2.2);
+        movie.setPrice(7.99);
+        movie.setCountries(countries);
+        movie.setGenres(genres);
+        movie.setReviews(reviews);
+
+        when(movieService.getMovieById(anyInt())).thenReturn(movie);
+
+        mockMvc.perform(get("/v1/movie/3"))
```

Comment:
looks like you don't move version parameter v1 to web.xml:
    <servlet-mapping>
        <servlet-name>movieland</servlet-name>
        <url-pattern>/v1/*</url-pattern>
    </servlet-mapping>